### DISCUSSION TITLE
VTable integrate with bokeh

### DISCUSSION THREAD

**Discussion by fangsmile** [bokeh/bokeh#13971] [Internals and design (Q&A)]
### Problem description

It is expected that VTable can be integrated, and VTable table library has a strong visual display ability

### Feature description
You can be viewed on the official website：https://visactor.io/vtable
![img_v3_029n_c20f8bf0-914d-480e-9369-9541ff50bc7g](https://github.com/bokeh/bokeh/assets/3468473/bbb68d0a-f6ac-4783-9df7-650e5ec26d5e)

### Potential alternatives

no

**Comment by bryevdv** [comment]
@fangsmile replacing the current data table or adding new different option alongside the current one would have a large cost, both up-front and in terms of maintenance. A proposal for such a large request needs to be much, *much* more thorough. Questions for you to consider:

* Are you also proposing to replace the current table implementation entirely, keeping the API exactly as it is? Or something new and separate with a different API? It's not clear what all you would want out of this ask
* What are the unique features of this new table that are better than, or missing, in the current SlickGrid-based model?
  * Are those things possible with SlickGrid, but simply missing in the Bokeh implementation? 
  * Adding/improving the existing implementation will most likely be less work to achieve. 
*  Conversely, are there features of the current SlickGrid table that *cannot* be achieved with the new table?
    
    This is a *very* important question to determine up-front. If this is a replacement, then has to support all the current features, or else a proper plan for deprecation, etc. would need to be considered. 
* Since you are evidently familiar with this other table library, are you available to help maintain a new table implementation in Bokeh? 
* What is the size cost of this new library? Is it modular? Bokeh itself has the table bundle split off separately but these are still important considerations. 

For a feature request like this, a proposal needs to be detailed and specific.

**Comment by fangsmile** [comment]
Ok！Then I will sort out the comparison between the two table library, and what are the benefits for us if we integrate this component VTable

**Comment by bryevdv** [comment]
Absent a more concrete proposal, moving to a discussion for now
